cypress/integration/email-error-test-create-new-assessment.js

```diff
@@ -0,0 +1,34 @@
+describe("Validates error on email creation failure", function () {
+  const email = Cypress.env("email");
+  const password = Cypress.env("password");
+  const baseUrl = Cypress.env("baseUrl");
+  const viewportWidth = Cypress.env("viewportWidth");
+  const viewportHeight = Cypress.env("viewportHeight");
+
+  // New assessment vars
+  const startNewButton = ".buttons > :nth-child(1) > .button-inner";
+  const addNewEmailButton = ".member-inputs > .desktop";
+  const errorMessage = ".error-message";
+
+  const beginNewAssesment = () => {
+    cy.get(startNewButton).click();
+  };
+
+  beforeEach(() => {
+    cy.visit(baseUrl);
+    cy.login(email, password);
+    cy.viewport(viewportWidth, viewportHeight);
+  });
+
+  it("finds proper error message on submit empty email", () => {
```

Comment:
missing test case on duplicated emails

Reply:
This might be a new ticket as the code to prevent this is not implemented...